<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="KOKATE-SR" failures="2" tests="2" name="Default test(failed)" time="1.052" errors="0" timestamp="2022-03-23T00:46:35 IST">
  <testcase classname="failed_xml_testNG.Failed_xml_Study" name="TC2" time="1.025">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method failed_xml_testNG.Failed_xml_Study.TC2() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failed_xml_testNG.Failed_xml_Study.TC2() didn't finish within the time-out 1000
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- TC2 -->
  <testcase classname="failed_xml_testNG.Failed_xml_Study" name="TC4" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failed_xml_testNG.Failed_xml_Study.TC4(Failed_xml_Study.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC4 -->
  <testcase classname="failed_xml_testNG.Failed_xml_Study" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
  <testcase classname="failed_xml_testNG.Failed_xml_Study" name="TC3" time="0.0">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase classname="failed_xml_testNG.Failed_xml_Study" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Default test(failed) -->
